spec/lib/pivotal-tracker/project_spec.rb

```diff
@@ -27,14 +27,14 @@
   describe "#find" do
     let(:project) { subject.find(project_id) }
     context "with a valid project id" do
-      let(:project_id) { 1962995 }
+      let(:project_id) { ENV['PROJECT_ID'] }
 
       it "is a project" do
         expect(project).to be_a(PivotalTracker::Project)
       end
 
       it "has the right id" do
-        expect(project.id).to eq(project_id)
+        expect(project.id.to_s).to eq(project_id)
```

Comment:
@lucyconklin do you know why this coercion is needed after changing to a variable? Just curious.